survey.html

```diff
@@ -31,31 +31,57 @@
-                    <label for="no">No</label><br>
-                    <input type="radio" id="?" name="opinion" value="?">
-                    <label for="?">None of your Business</label><br>
-                    <label for="life">What is the meaning of life?:</label>
-                    <textarea id="life" name="life" rows="5" cols="50"></textarea>
-                    <input type="submit" value="Submit">
+                    <p>
+                        <label for="fname">First Name: <abbr title="required" aria-label="required">*</abbr></label>
+                        <input type="text" id="fname" name="name_field" required>
+                    </p>
+                    <p>
+                        <label for="email">Enter your email: <abbr title="required" aria-label="required">*</abbr></label>
+                        <input type="email" id="email" name="email_field" required>
+                    </p>
+                    <p>
+                        <label for="quantity">What is your favourite number?</label>
+                        <input type="number" id="quantity" name="best_whole_number" min="1" max="999999999">
+                    </p>
+                    <p>
+                        <label for="day">Which day is best?</label>
+                        <input type="date" id="day" name="best_day">
+                    </p>
+                    <p>
+                        <label for="bear">Which bear is best?<abbr title="required" aria-label="required">*</abbr></label>
+                        <select name="best_bear" id="bear" required>
+                            <option value="">&nbsp;</option>
+                            <option value="Black">Black Bear</option>
+                            <option value="Grizzly">Grizzly Bear</option>
+                            <option value="Polar">Polar Bear</option>
+                            <option value="Sun">Sun Bear</option>
+                            <option value="Moon">Moon Bear</option>
+                        </select>
+                    </p>
+                    <fieldset>
+                        <legend>Do you like radio buttons?</legend>
+                        <ul>
+                            <li>
+                                <input type="radio" id="yes" name="opinion" value="yes">
+                                <label for="yes">Yes</label>
+                            </li>
+                            <li>
+                                <input type="radio" id="no" name="opinion" value="no">
+                                <label for="no">No</label>
+                            </li>
+                            <li>
+                                <input type="radio" id="?" name="opinion" value="?" checked>
+                                <label for="?">None of your Business</label>
+                            </li>
+                        </ul>
+                    </fieldset>
```

Comment:
Radio buttons were in and unordered list inside a fieldset as per lab instructions